src/icons/OrbBox.tsx

```diff
@@ -1,34 +1,34 @@
-import Colors, { colorToDropShadow } from 'models/Colors'
+import Color from 'models/Color'
 import classnames, {
-  TDropShadow,
-  dropShadow,
   fill,
   opacity,
   transitionProperty,
 } from 'classnames/tailwind'
+import colorToDropShadow from 'helpers/colorToDropShadow'
+import colorToStrokeColor from 'helpers/colorToStrokeColor'
 
-const orbBoxStyles = (shadowColor: TDropShadow, shadow?: boolean) =>
+const orbBoxStyles = (shadow?: boolean) =>
   classnames(
-    dropShadow(shadowColor),
     opacity(shadow ? 'opacity-100' : 'opacity-90'),
     transitionProperty('transition-all')
   )
 const fillPath = fill('fill-primary-dark')
 
 interface OrbBoxProps {
-  color: Colors
+  color: Color
   shadow?: boolean
```

Comment:
This seems like a misnomer. Should it be named `isShadowOpaque`?

Reply:
renamed it